tyr/clusters/mongo.py

```diff
@@ -54,6 +54,14 @@ def provision(self):
 
             zones += zones
 
+        if len(self.subnet_ids) < 1:
```

Comment:
Just a thought. The length of a list will never be negative, so the only possible case where

```python
len(self.subnet_ids) < 1
```

is `True` is when the list is empty. It might be better to write it as

```python
if len(self.subnet_ids) == 0
```

so you don't have to take the time to think about all the cases where it would be `True`.